csharp-intermediate-topics/RenameFilesWithCSharp/RenameFilesWithCSharp/BatchRenameFiles.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace RenameFilesWithCSharp;
+
+public class BatchRenameFiles
```

Comment:
This whole class could be marked as `static`. Also, the name of the class isn't the best. It is using verb in its name, which is normally applied to methods, not classes. Why not call it something like `FileNameService` or `FileProcessor`?